packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -668,6 +669,19 @@ class _FocusableActionDetectorState extends State<FocusableActionDetector> {
   }
 }
 
+/// Creates the default map of actions used by [WidgetsApp] to define the set
+/// of available actions for an application.
+Map<LocalKey, ActionFactory> defaultActions(TargetPlatform targetPlatform) {
+  // At the moment, all platforms have the same set of available actions.
+  return <LocalKey, ActionFactory>{
+    DoNothingAction.key: () => const DoNothingAction(),
+    RequestFocusAction.key: () => RequestFocusAction(),
+    NextFocusAction.key: () => NextFocusAction(),
+    PreviousFocusAction.key: () => PreviousFocusAction(),
+    DirectionalFocusAction.key: () => DirectionalFocusAction(),
+  };
+}
+
 /// An [Action], that, as the name implies, does nothing.
```

Comment:
Down below, does the documentation for `ActivateAction` and `SelectAction` need to be changed, re: `SelectAction` mentions checkboxes and radio buttons?